src/app/main-page/main-page.component.html

```diff
@@ -1 +1,18 @@
-<h1>Change me</h1>
\ No newline at end of file
+<div class="example-container ml-3">
+    <mat-form-field appearance="fill">
+      <mat-label>Input</mat-label>
+      <input matInput>
+    </mat-form-field>
+    <br>
+    <mat-form-field appearance="fill">
+      <mat-label>Select</mat-label>
+      <mat-select>
+        <mat-option value="option">Option</mat-option>
+      </mat-select>
+    </mat-form-field>
+    <br>
+    <mat-form-field appearance="fill">
+      <mat-label>Textarea</mat-label>
+      <textarea matInput></textarea>
+    </mat-form-field>
+</div>
```

Comment:
This page now correctly displays some example forms from Angular Material.

Reply:
Excellent, I will use these examples for the form page